frame/session/src/historical.rs

```diff
@@ -71,6 +71,20 @@ decl_module! {
 		fn on_initialize(_n: T::BlockNumber) {
 			CachedObsolete::<T>::remove_all();
 		}
+		fn on_runtime_upgrade() {
+			migration::migrate::<T>();
+		}
+	}
+}
+
+mod migration {
+	use super::*;
+	pub fn migrate<T: Trait>() {
+		if let Some((begin, end)) = StoredRange::get() {
```

Comment:
@rphmeier please confirm this is correct.

Reply:
Looks fine